src/middleware/__tests__/apiKeyChecker.unit.test.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs';
+import { AuthenticationError } from '../../errors';
+import { checkApiKey } from '../apiKeyChecker';
+import type { NextFunction, Request, Response } from 'express';
+
+describe('Authorization middleware', () => {
+  let mockRequest: Partial<Request> = {};
```

Comment:
Is there value in defining this outside the scope it is used?

I'm a little worried that this creates the possibility for odd outcomes when we break out of bands / start running things in parallel though I don't know if that worry is justified.